src/modules/app/components/LaunchInquiry.js

```diff
@@ -59,6 +59,19 @@ class LaunchInquiry extends React.Component<PropsType, AppStateType> {
       } else {
         this.setState({waitingForSentry: false})
```

Comment:
Do we need to only set this after the other inquiries? Or is this fine?

Reply:
It is fine to set this as soon as sentry is allowed.